core/arch/arm/instr.c

```diff
@@ -883,3 +883,17 @@ instr_is_exclusive_store(instr_t *instr)
     return (opcode == OP_strex || opcode == OP_strexb || opcode == OP_strexd ||
             opcode == OP_strexh);
 }
+
+DR_API
+bool
+instr_is_scatter(instr_t *instr)
+{
+    return false;
```

Comment:
Not sure about this one -- did you check somehow?

Reply:
I am pretty sure yes, but could add a XXX and ask the ARM guys?